### PR TITLE
Fix default handling to be as implied. Closes #1663

### LINKED ISSUE #1663: When disabling a client with "pivpn -off", you must explicitly type "y" for the disablement to occur
## In raising this issue, I confirm the following:

`{please fill the checkboxes, e.g: [X]}`

- [X] I have read the [documentation](https://docs.pivpn.io)
- [] Is it a feature request? please consider opening a [Discussion] (https://github.com/pivpn/pivpn/discussions/new?category=feature-requests)
- [] Is this a support request? please consider copying the content of this document and using it to open a [Discussion](https://github.com/pivpn/pivpn/discussions/new?category=support)
- [] I have read and understood the [PiVPN General Guidelines](https://github.com/pivpn/pivpn/#general-guidelines)
- [X] The issue I am reporting can be *replicated*.  
- [] The issue I am reporting *is* directly related to the pivpn installer script.
- [X] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pivpn/pivpn/wiki/FAQ), [closed issues](https://github.com/pivpn/pivpn/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aclosed), and [open issues](https://github.com/pivpn/pivpn/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aopen)).

### Has your install failed? 
```
no
```

### Describe the issue

When disabling a client with "pivpn -off", you must explicitly type "y" for the disablement to occur.  **The capital "Y", suggesting the default action that the selected client will get disabled by just hitting Enter, is not true.**

**Expected behavior**

If you offer me a capital Y, suggesting a default "yes", and I accept that by hitting Enter, then disable the client!  I only caught the error by going back and checking with "pivpn -l".  No disablement occurred!  

Then I did the "pivpn -off" again, this time giving explicit "y" when asked.  Then the disablement truly occured when checked with "pivpn -l".

**Screenshots**

### Can you replicate the issue? Describe the steps below

```
user@host:~$ pivpn -off                                                                                   
::  Client list  ::                                                                                      
1) client1                                                                                                
2) client2                                                                                                  
Please enter the Index/Name of the Client to be removed from the list above: 2                           
Confirm you want to disable test? **[Y/n]**          _...and just hit enter here, trusting a default "yes"_                                                        
vadm@v2c:~$ pivpn -l                                                                                     
::: Clients Summary :::
Client      Public key                                        Creation date
client1      blahblah1=      01 Dec 2022, 09:39, UTC
client2        blahblah2=      02 Dec 2022, 07:04, UTC
::: Disabled clients :::
user@host:~$ 
```
**No disabled clients are seen above, under the "::: Disabled clients :::"!**

### Have you searched for similar issues and solutions?

yes

**Additional context**

Running on Debian 11

### Have you taken any steps towards solving your issue?
```
 no
```

#### What type of hardware are you running PiVPN at?

```
VPS server in cloud
```
#### Output of `uname -a`
```
Linux host 5.10.0-19-amd64 #1 SMP Debian 5.10.149-2 (2022-10-21) x86_64 GNU/Linux
```

#### Output of `cat /etc/os-release`
```
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

### PR DESCRIPTION
**Pull request by CoolCat467**
Make user input handling look for blanks so that implied defaults work properly.
This fixes #1663, which is correct in that when you have a user input question and you give the options Y/n or y/N, it is implied that the capitalized letter is what the program will default to if there is no input.

### CODE REVIEW

**Comment by 4s3ti** [comment]
Thank you for your PR ... Please read: https://github.com/pivpn/pivpn#pull-requests

File: scripts/wireguard/disableCONF.sh [hunk 1]
```diff
@@ -106,7 +106,7 @@ for CLIENT_NAME in "${CLIENTS_TO_CHANGE[@]}"; do
       read -r -p "Confirm you want to disable ${CLIENT_NAME}? [Y/n] "
     fi
 
-    if [[ "${REPLY}" =~ ^[Yy]$ ]]; then
+    if [[ "${REPLY}" =~ ^[yY]$ ]] || [[ -z "${REPLY}" ]]; then
```
Comment (orazioedoardo) [inline scripts/wireguard/disableCONF.sh:109]: Why inverting `y` and `Y` though?

Comment (CoolCat467) [reply scripts/wireguard/disableCONF.sh:109]: Mostly because I incorrectly reverted a change I made, I can switch it back

Comment (CoolCat467) [reply scripts/wireguard/disableCONF.sh]: Fixed in 90ba2d1

**Review by 4s3ti — CHANGES_REQUESTED** [review]
Too many unnecessary Commits which are also not following the commit style. 

Please squash your commits and amend the commit messages to follow the guidelines here: https://github.com/pivpn/pivpn#pull-requests.

**Comment by CoolCat467** [comment]
Re-opening as separate pull request

**Pull request closed without merging**